docs/ipfs/security.md

```diff
@@ -0,0 +1,19 @@
+# Security
+
+### RPC nodes
+The IPFS build utilizes non-secret environment variables since all IPFS content must be accessible to anyone.
+Therefore, the widget uses public RPC nodes to serve RPC requests. Users are explicitly notified about this fact in the UI,
+allowing them an option to specify necessary RPC nodes on the settings page. RPC nodes setup will be stored in a browser's localStorage and used for subsequent visits to the same IPFS gateway.
+
+### Possible localStorage leak (important!)
+Lido widgets use your browser's localStorage to store some UI settings and RPC nodes urls.
+If you are using an IPFS gateway, which is referencing CID hash as a part of the URL path (e.g., `{GATEWAY_DOMAIN}/ipfs/{HASH}`),
+rather than the subdomain (e.g., `{HASH}.{GATEWAY}`), then other websites accessed from the same IPFS gateway
+can potentially view or edit your settings, because localStorage stays the same for the same domain.
+
+To avoid this possibility, we suggest using IPFS gateway URL, attached to the IPFS release description,
```

Comment:
To avoid this possibility, it is suggested to use IPFS gateway URL, attached to the IPFS release description